release-notes.md

```diff
@@ -327,6 +334,77 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v39.0
+
+This release has been tracked in our [roadmap
+project](https://github.com/orgs/cloud-hypervisor/projects/6) as iteration
+v39.0. The following user visible changes have been made:
+
+### Variable Sizing of PCI Apertures for Segments
+
+It is now possible to use `--pci-segment` to adjust the aperture size that
+devices 32-bit and 64-bit PCI device BARs will be allocated from. Previously
+the address space was equally distributed across all the segments which may
+leave insufficient space for devices that require a large 32-bit space. With
+this change the weighting per segment can be adjusted. (#6387)
+
+### Direct Booting with bzImages
+
+Support for directly booting Linux from bzImages has been added.(#6200)
+
+### Support for NVIDIA GPUDirect P2P Support
+
+The `x_nv_gpudirect_clique` option was added to `--device` to allow the
+configuration of device P2P support with NVIDIA GPUs. (#6235)
+
+### Guest NMI Injection Support
+
+A new API endpoint and `ch-remote` option added for injecting an NMI into the
+guest. (#6047)
+
+### Notable Bug Fixes
+
+* Workaround for kernel bug affecting guest IRQ masking on AMD (#6353)
+* Correctly cleanup `sigwinch_listener` process (#6208)
+* Graceful shutdown of HTTP API thread (#6248, #6247)
+* Fix `queue_affinity` option in OpenAPI metadata (#6268)
+* Fix documentation to indicate only stream mode is supported by `virtio-vsock`
+  (#6306)
+* Fix `virtio-fs` tag validation (#6358, #6359)
+* Add missing `pvpanic` device to OpenAPI metadata (#6372)
+* Fixes for nested virtualization with VFIO devices (#6110, #6298, #6297,
+  #6319)
+* Fixes for backing file for `virtio-mem` regions with snapshot/restore (#6337,
```

Comment:
Nit: better to stay with imperative tone, say `\sFixes for\Fix`

Reply:
Ah, one of them was wrong - but the nested virtualization one is plural.